.github/workflows/wheel.yml

```diff
@@ -21,7 +21,7 @@ concurrency:
 
 jobs:
   build_wheels_matrix:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
pinning OS versions in `wheel.yml` hoping to get less unexpected failues in CI (due to edits to the runner image like removing libmagic)